_includes/component_url

```diff
@@ -1,16 +1,20 @@
 {%- if release.title != "master" -%}
 {%- case include.component -%}
+{%- comment -%}
+note that all 'or's in the below case statement are there to maintain backwards compatability 
+with how image names used to be referenced in config.yml
+{%- endcomment -%}
 
-{%- when 'calico/node' -%}
+{%- when 'node' or 'calico/node' -%}
```

Comment:
Are you checking both the old and new key values in order to maintain upgrade path?

Reply:
yup. the comment I added above was supposed to explain that :smile:

Reply:
oops, yeah.